### PR TITLE
Updates to Thanos Migration guide

### PR DESCRIPTION
**Pull request by mclean0328**
## Proposed Changes
Updates to make the document more explicit.

### CODE REVIEW

File: install-and-configure/install/multi-cluster/federated-etl/thanos-migration-guide.md [hunk 1]
```diff
@@ -26,11 +26,15 @@ Important notes for the migration process:
 
 ## Migration process
 
-All of these steps should be performed on Kubecost v1.108.1. Only at the end, will you upgrade to Kubecost 2.0. The goal of this doc is to gradually migrate off Thanos (it is no longer supported in the Kubecost v2.0 Helm chart), towards Federated ETL, then finally Aggregator.
+All of these steps should be performed on Kubecost v1.108.1 on the Primary Kubecost cluster. Only at the end, will you upgrade all Kubecost clusters to Kubecost 2.0. The goal of this doc is to gradually migrate off Thanos (it is no longer supported in the Kubecost v2.0 Helm chart), towards Federated ETL, then finally Aggregator.
+
+Please upgrade to v1.108.1 on the Primary before completing any of the following steps. v1.108.1 ships with various utilities necessary to complete the upgrade to v2.0
```
Comment (thomasvn) [inline install-and-configure/install/multi-cluster/federated-etl/thanos-migration-guide.md:31]:
```suggestion
Please upgrade to v1.108.1 on the Primary before completing any of the following steps. v1.108.1 ships with various utilities necessary to complete the upgrade to v2.0.
```

File: install-and-configure/install/multi-cluster/federated-etl/thanos-migration-guide.md [hunk 2]
```diff
@@ -112,6 +118,10 @@ Ensure all data loads into the Kubecost UI before moving onto Step 7.
 
 ### Step 7: Upgrade your secondary clusters to build and push ETL data
 
+For this step, the secondary clusters DO NOT need to be on v2.0.  However, you must be running a version of Kubecost that supports Federated ETL.  >v1.99.0.
```
Comment (thomasvn) [inline install-and-configure/install/multi-cluster/federated-etl/thanos-migration-guide.md:121]:
```suggestion
For this step, the secondary clusters **DO NOT** need to be on v2.0. However, you must be running a version of Kubecost that supports Federated ETL (greater than v1.99.0).
```

File: install-and-configure/install/multi-cluster/federated-etl/thanos-migration-guide.md [hunk 3]
```diff
@@ -135,9 +145,28 @@ Optionally, you can remove the [Thanos sidecar](https://raw.githubusercontent.co
...
+
+```sh
+helm upgrade kubecost cost-analyzer --repo https://kubecost.github.io/cost-analyzer/ \
+  --namespace kubecost \
+  -f values.yaml
+```
+
+### Step 10: Upgrade Secondary Clusters to Kubecost 2.0 (Optional)(Recommended)
+
+*While not absolutely necessary to upgrade secondary clusters to 2.0 immediately, we do recommend doing so as soon as possible
```
Comment (thomasvn) [inline install-and-configure/install/multi-cluster/federated-etl/thanos-migration-guide.md:160]:
```suggestion
{% hint style="info" %}
While not absolutely necessary to upgrade secondary clusters to 2.0 immediately, we recommend doing so as soon as possible.
{% endhint %}
```

File: install-and-configure/install/multi-cluster/federated-etl/thanos-migration-guide.md [hunk 4]
```diff
@@ -26,11 +26,22 @@ Important notes for the migration process:
...
+All of these steps should be performed on Kubecost v1.108.1 on the primary Kubecost cluster. Only after completing the steps in this guide should you upgrade all Kubecost clusters to v2.0. The goal of this doc is to gradually migrate off Thanos, which is no longer supported in the Kubecost v2.0+ Helm chart, towards Federated ETL, then finally Aggregator.
+
+Please upgrade your primary cluster to v1.108.1 before completing any of the following steps. v1.108.1 ships with various utilities necessary to complete the upgrade to v2.0. You can upgrade to this specific version using this following command:
+
+```
+helm upgrade --install kubecost \
+  --repo https://kubecost.github.io/cost-analyzer/ cost-analyzer \
+  --namespace kubecost --create-namespace \
+  --version 1.108.1
+```
```
Comment (bstuder99) [inline install-and-configure/install/multi-cluster/federated-etl/thanos-migration-guide.md:38]: I've added this section showing how to upgrade your Kubecost to a specific version, can someone verify this is correct?

Comment (bstuder99) [reply install-and-configure/install/multi-cluster/federated-etl/thanos-migration-guide.md:38]: Not sure if they need to create the Kubecost namespace if it already exists

File: install-and-configure/install/multi-cluster/federated-etl/thanos-migration-guide.md [hunk 5]
```diff
@@ -26,11 +26,22 @@ Important notes for the migration process:
...
 
-All of these steps should be performed on Kubecost v1.108.1. Only at the end, will you upgrade to Kubecost 2.0. The goal of this doc is to gradually migrate off Thanos (it is no longer supported in the Kubecost v2.0 Helm chart), towards Federated ETL, then finally Aggregator.
+All of these steps should be performed on Kubecost v1.108.1 on the primary Kubecost cluster. Only after completing the steps in this guide should you upgrade all Kubecost clusters to v2.0. The goal of this doc is to gradually migrate off Thanos, which is no longer supported in the Kubecost v2.0+ Helm chart, towards Federated ETL, then finally Aggregator.
+
+Please upgrade your primary cluster to v1.108.1 before completing any of the following steps. v1.108.1 ships with various utilities necessary to complete the upgrade to v2.0. You can upgrade to this specific version using this following command:
+
+```
+helm upgrade --install kubecost \
+  --repo https://kubecost.github.io/cost-analyzer/ cost-analyzer \
+  --namespace kubecost --create-namespace \
```
Comment (thomasvn) [inline install-and-configure/install/multi-cluster/federated-etl/thanos-migration-guide.md:36]:
```suggestion
  --namespace kubecost \
```

**Review by thomasvn — APPROVED** [review]
After the final edit on the `helm upgrade` command, this LGTM!

**Review by bstuder99 — APPROVED** [review]

**Pull request merged**